prow/config/jobs/istio.yaml

```
@@ -234,6 +234,20 @@ jobs:
     - prow/config/ambient-sc.yaml
     - test.integration.ambient.kube
 
+  - name: integ-ambient-owned-cni
+    modifiers: [presubmit_optional]
+    optional: true
```

this change broke the test.
```suggestion
    optional: true
```

---

it shouldn't break the test, how did it do that?

---

Go Docs showing the field: https://pkg.go.dev/sigs.k8s.io/prow/pkg/config#Presubmit, we really shouldn't add a non-optional test without knowing it will work because it will block PRs from merging. We did this just a couple weeks ago with Keith's ambient-mc integ tests